doc/lua_api.txt

```diff
@@ -4624,6 +4624,16 @@ Utilities
 * `minetest.mkdir(path)`: returns success.
     * Creates a directory specified by `path`, creating parent directories
       if they don't exist.
+* `minetest.rmdir(path, recursive)`: returns success.
+    * Removes a directory specified by `path`.
+    * If `recursive` is set to `true`, the directory is recursively removed.
+      Otherwise, the directory will only be removed if it is empty.
+* `minetest.cpdir(source, destination)`: returns success.
+    * Copies a directory specified by `path` to `destination`, `destination` is
+      overwritten if it already exists.
+* `minetest.mvdir(source, destination)`: returns success.
+    * Moves a directory specified by `path` to `destination`, `destination` is
```

Comment:
This is incorrect, the call errors if the destination already exists.

Reply:
ah, I misread the code - it uses rmdir, which only works with empty dirs